src/qt/sendcoinsdialog.cpp

```diff
@@ -311,72 +306,82 @@ void SendCoinsDialog::on_sendButton_clicked()
         formatted.append(recipientElement);
     }
 
-    QString questionString;
     if (model->wallet().privateKeysDisabled()) {
-        questionString.append(tr("Do you want to draft this transaction?"));
+        question_string.append(tr("Do you want to draft this transaction?"));
     } else {
-        questionString.append(tr("Are you sure you want to send?"));
+        question_string.append(tr("Are you sure you want to send?"));
     }
 
-    questionString.append("<br /><span style='font-size:10pt;'>");
+    question_string.append("<br /><span style='font-size:10pt;'>");
     if (model->wallet().privateKeysDisabled()) {
-        questionString.append(tr("Please, review your transaction proposal. This will produce a Partially Signed Bitcoin Transaction (PSBT) which you can copy and then sign with e.g. an offline %1 wallet, or a PSBT-compatible hardware wallet.").arg(PACKAGE_NAME));
+        question_string.append(tr("Please, review your transaction proposal. This will produce a Partially Signed Bitcoin Transaction (PSBT) which you can save or copy and then sign with e.g. an offline %1 wallet, or a PSBT-compatible hardware wallet.").arg(PACKAGE_NAME));
```

Comment:
- s/Please,/Please/
- s/wallet,/wallet or/
- could omit "e.g."

(these were not introduced by this PR, can ignore)